web/package.json

```diff
@@ -13,7 +13,6 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
-    "typescript": "^4.9.5",
```

Comment:
@sanghyun1128 FYI. https://github.com/facebook/create-react-app/issues/6180#issuecomment-453639887 if you moved this to `devDependencies` for some reason

Reply:
There was no particular reason, but according to the link, it doesn't seem to be a problem. So since it's already been moved, is it okay to leave it as it is? or restore it.

Reply:
Sure why not! Just for your information